relnotes/smartunion.feature.md

```diff
@@ -0,0 +1,5 @@
+* `ocean.core.SmartUnion`
+
+  Helper function `handleInvalidCases` in `SmartUnion` is added as the support
```

Comment:
It's a plain mixin string now (not a function any more).

Reply:
Ah, I forgot to update it here (already updated commit message).